controllers/onload_controller.go

```diff
@@ -143,8 +155,8 @@ func (r *OnloadReconciler) Reconcile(
 }
 
 const kmmModuleLabelPrefix = "kmm.node.kubernetes.io/version-module"
-const onloadModuleNameSuffix = "-onload-module"
-const sfcModuleNameSuffix = "-sfc-module"
+const onloadModuleNameSuffix = "-module"
```

Comment:
My thought is that we would need some way to to tell the difference between the onload module and the potential sfc module and I think that a name suffix is a good way to do that. Additionally I thought it would be good if the names of the onload module and the sfc module are the same length which would simplify the logic when checking if the name is too long.